chat-plugins/shop.js

```diff
@@ -486,14 +444,9 @@ exports.commands = {
 	rs: 'resetsymbol',
 	resetsymbol: function(target, room, user) {
 		if (!user.hasCustomSymbol) return this.sendReply('You don\'t have a custom symbol!');
-		user.getIdentity = function() {
-			if (this.muted) return '!' + this.name;
-			if (this.locked) return '‽' + this.name;
-			return this.group + this.name;
-		};
-		user.hasCustomSymbol = false;
-		delete user.getIdentity;
```

Comment:
Check the lines I added after this. <code>user.getIdentity = Users.User.prototype.getIdentity;</code> resets the definition of <code>user.getIdentity()</code> to what it's originally supposed to be (we change the definition of this in the customsymbol command).